psi4/driver/procrouting/proc.py

```diff
@@ -2217,6 +2217,9 @@ def run_scf_hessian(name, **kwargs):
     H = core.scfhess(ref_wfn)
     ref_wfn.set_hessian(H)
 
+    # Clearly, add some logic when the reach of this fn expands
+    ref_wfn.set_variable('HF TOTAL HESSIAN', H)
```

Comment:
It's `HF` here, and `SCF` in a different file below.  Is that intentional?  Also, is the intent of the `TOTAL` qualifier to eventually allow reporting of correlation contributions separately?

Reply:
Yes, the idea is that `SCF TOTAL HESSIAN` gets set where the code knows is an SCF (in scfhess) and `HF TOTAL HESSIAN` gets set where the code knows it's Hartree–Fock. Then `B3LYP TOTAL HESSIAN` would get set for that method.

The "total" is in analogy to `MP2 TOTAL ENERGY`, where ref and corl separation is widespread. Not expecting any such for grad/hess, unless `HF TOTAL HESSIAN` was separately available.

Reply:
Ok, just wanted to check to make sure about the SCF/HF terminology.  I have similar reservations to you about the separation of grad/hess components, but I like the idea of consistent key for the data arrays.  Thanks for the clarification.